src/ngrams_generator/ngrams_generator.py

```diff
@@ -106,23 +115,91 @@ def detect_language(self, entry: str)->str:
         except Exception as e:
             logging.warning(f"{e} Using english(en) instead.")
         return "en"
-    
+
+    def handle_currency(self, entry_lst: List)-> List:
+        """
+        Handles currency
+        """
+        flag = 0
```

Comment:
Please rename `flag` to something informative like `is_previous_token_added` or something like that.

Reply:
@bewakes  this is done.